### ISSUE TITLE
Acala and Karura support

### ISSUE THREAD

**Issue by darko-mijic** [stylo-app/stylo#17]
Hello,

I have tried using Karura and Acala (Mandala testnet) apps with Stylo but I am unable to sign transactions. I am getting `Unable to parse QR data` errors. 

Are there plans to support Acala and Karura app transactions?

**Comment by Tbaut** [comment]
Acala and Karura are supported as of v0.9.9

**Comment by darko-mijic** [comment]
The current production Acala app is not supported yet? 
I've tried claiming ACA and got this error. I did not proceed with the transaction.

![image](https://user-images.githubusercontent.com/4280521/150976534-f11b0fa5-d93b-42e5-8dd2-c240b64458c0.png)

**Comment by Tbaut** [comment]
Yes. So Stylo support Acala and Karura, that's why this issue is closed. Now it needs to be kept updated every time there is a runtime upgrade, which may happen often.

Stylo is a community maintained app, any PR is welcome to update to the latest Acala metadata :)

**Comment by GalioColossus** [comment]
@Tbaut it there any documentation how to update the metadata? I would like to help

**Comment by Tbaut** [comment]
Now because Acala released its token, I will push an update in the next hours. Keep an eye on the releases :)

**Comment by Tbaut** [comment]
it's been published. https://github.com/stylo-app/stylo/releases/tag/v0.9.11
